src/sider/frisktilarbeid/FerdigbehandletVedtak.tsx

```diff
@@ -0,0 +1,74 @@
+import React, { ReactElement, useState } from "react";
+import { Alert, BodyLong, Box, Button, Heading } from "@navikt/ds-react";
+import dayjs from "dayjs";
+import { tilDatoMedManedNavn } from "@/utils/datoUtils";
+import { FattNyttVedtak } from "@/sider/frisktilarbeid/FattNyttVedtak";
+import { VedtakResponseDTO } from "@/data/frisktilarbeid/frisktilarbeidTypes";
+
+const texts = {
+  heading: "Start nytt vedtak",
+  forrigeVedtakInfo: (
```

Comment:
Foretrekker å sende inn et objekt i stedet for mange parametre som må "passes" i riktig rekkefølge... I dette tilfellet kunne man kanskje bare sendt inn `vedtak`? :thinking:

Reply:
Vi må i så fall gjøre alt av dato aformatering her, det tror jeg ikke blir så pent😬 Enig i at det ble litt mange input parametere her da

Reply:
Hvis jeg forsår deg riktig her da

Reply:
Hmja, det var tanken. Man trenger vel bare dato-formateringen til tekstene så tenkte det var like pent om det gjøres der. Evt om vi lar `texts` bare inneholde rene tekster og så kan vi ha en egen funksjon inni komponenten for å bygge opp den `forrigeVedtakInfo`-teksten. Tror sistnevne er mer på linje med hvordan vi gjør det andre steder.